frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -1136,6 +1143,7 @@ export const widgets = [
         visibility: { value: '{{true}}' },
         disabledState: { value: '{{false}}' },
         borderRadius: { value: '{{0}}' },
+        backgroundColor: { value: '' },
```

Comment:
This adds the property value for **PasswordInput**
https://github.com/ToolJet/ToolJet/blob/1dedef8f9ad0b51b5448741f38b70ecf2b3ef0e6/frontend/src/Editor/WidgetManager/widgetConfig.js#L1073-L1146

Can you also populate it with a default value instead of an empty string?

```suggestion
        backgroundColor: { value: '#ffffffff' }
```